spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
-[elsewhere](#security_considerations).
-
-The transformation begins by removing the "https://" prefix from this URL. If
-the path portion of the URL consists exactly of "/.well-known/did.json", this
-suffix is also removed. If the URL ends in a slash, the final slash is removed.
-All remaining slashes in the URL are replaced with colons. If the URL ends with
-".json", this suffix is deleted. If the URL includes a port, the colon that
-delimits the port number MUST be percent-encoded so colons in the MSI map only
-to slashes.
-
+The `did:webs` method specific identifier should be considered to have two
+parts, a fully qualified domain name with an optional path (identical to
+`did:web`), plus a KERI AID (autonomic identifier) that is always the final
+component of the path. A KERI AID is a unique-identifier derived from the
+inception event of a KERI identifier--the first item in the [[ref: KEL]]. Any
+KERI identifier (DIDs or AIDs) that reference the same AID must, by definition,
+refer to the same KEL, and hence, be the same identifier. An `did:webs` DIDs
+that have the same AID are by definition, synonyms of one another.
+
+The fully qualified domain name is secured by a TLS/SSL certificate with an
+optional path. The formal rules describing valid domain name syntax are
+described in [RFC1035], [RFC1123], and [RFC2181]. The fully qualified domain
+name MUST match the common name used in the SSL/TLS certificate, and MUST NOT
+include IP addresses. A port MAY be included and the colon MUST be percent
+encoded to prevent a conflict with paths. Directories and subdirectories MAY
+optionally be included, delimited by colons rather than slashes.
+
+After the fully qualified domain name and optional path in a `did:webs` is a
+final path component, a colon and the AID. To be compatible with `did:web`, the
+AID is "just a path", the final (and perhaps only) path element. The presence of
+the required AID as a path element means that a `did:webs` always has a path,
+and so the "no path" version of a `did:web` that implicitly uses the
+`.well-known` folder is not supported by `did:webs`. Any `did:webs` can be
+expressed as a `did:web` but the inverse is not true--a `did:webs` MUST include
+an AID.
+
+As with `did:web`, the DIDDoc is found by transforming the DID to an
+HTTPS URL as follows:
+
+- Replace `did:webs` with `https://`
+- Replace the "`:`"s in the method-specific identifier with path separators, "'/'"s
+- Convert the optional port percent encoding ("`%3A`"`) to a colon.
+- Append "`/did.json`" to the resulting string.
+
+A GET on that URL MUST return the DIDDoc.
+
+Another transformation produces a second URL that is used to retrieve the KERI KEL:
+
+- Replace the trailing "`/did.json`" with "`/did.kel`".
+
+A GET on that URL MUST return the KEL for the AID in the `did:webs` identifier.
+
+After retrieval, the KEL SHOULD BE processed using KERI rules by anyone
+resolving the DIDDoc to verify it's contents. Further, KERI-aware applications
```

Comment:
"it's" --> "its"

Reply:
If someone doesn't verify the contents against the KEL, we have a serious security problem. I'm tempted to require processing against the KEL. Needs discussion.